dotenv.cabal

```diff
@@ -134,6 +135,7 @@ test-suite dotenv-test
                      , exceptions >= 0.8 && < 0.11
                      , hspec-megaparsec >= 2.0 && < 3.0
                      , mtl
+                     , data-default-class >= 0.1.2 && < 0.2
```

Comment:
Is it necessary to add it to the test-suite?

Reply:
I think it is, because the test suite specifies the src directory` in `hs-source-dirs`.

Reply:
Oh, you're right. I'll fix that in a separated PR. Thanks man!

Reply:
I thought it was on purpose haha you're welcome :)